spec/openapi.yaml

```diff
@@ -1132,6 +1163,60 @@ components:
           description: Package version available for update
           example: "4.2-RC1"
 
+    Transfer:
+      type: object
+      properties:
+        type:
+          type: string
+          enum: [ NO_TRANSFER, FROM_WEB, FROM_CONNECT, FROM_PRINTER, FROM_SLICER, FROM_CLIENT, TO_CONNECT, TO_CLIENT ]
+          example: FROM_WEB
+        display_name:
+          type: string
+          description: Long Filename
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        path:
+          type: string
+          example: "/local"
+        display_path:
```

Comment:
I thought we talked about `display_path` being necessary only in `v1/job`, but since it is not required I guess it is OK :man_shrugging: